src/io_restart.F90

```diff
@@ -305,10 +309,14 @@ subroutine create_new_file(id)
         do l=1, id%ndim ! list all defined dimensions 
            if (kdim==id%dim(l)%size) dimid(k)=id%dim(l)%code
         end do
-!________write(*,*) kdim, ' -> ', dimid(k)__________________________________
+        !write(*,*) "j",j,kdim, ' -> ', dimid(k)
      end do
-     id%error_status(c) = nf_def_var(id%ncid, trim(id%var(j)%name), NF_DOUBLE, id%var(j)%ndim+1, &
-                       (/dimid(1:n), id%rec/), id%var(j)%code); c=c+1
+     id%error_status(c) = nf_def_var(id%ncid, trim(id%var(j)%name), NF_DOUBLE, id%var(j)%ndim+1, (/dimid(1:n), id%rec/), id%var(j)%code); c=c+1
+     !if (n==1) then
+     !   id%error_status(c)=nf_def_var_chunking(id%ncid, id%var(j)%code, NF_CHUNKED, (/1/)); c=c+1 
+     if (n==2) then
+        id%error_status(c)=nf_def_var_chunking(id%ncid, id%var(j)%code, NF_CHUNKED, (/1, id%dim(1)%size/)); c=c+1 
```

Comment:
if you comment out (...!c=c+1)  then the error won't be collected in the error_status array. Then in case if NF_CHUNKED does not exist in the netcdf library the model most probably won't blowup (tested for ollie).

Reply:
same in io_meandata